libcxx/include/__functional/move_only_function_impl.h

```diff
@@ -253,8 +237,8 @@ class _LIBCPP_MOVE_ONLY_FUNCTION_TRIVIAL_ABI move_only_function<_ReturnT(
   _LIBCPP_HIDE_FROM_ABI friend bool operator==(const move_only_function& __func, nullptr_t) noexcept { return !__func; }
 
 private:
-  using _CallFn    = _ReturnT(_LIBCPP_MOVE_ONLY_FUNCTION_CV std::byte*, _ArgTypes...);
-  using _DestroyFn = void(void*);
+  using _CallFn    = _ReturnT(std::byte*, _ArgTypes...) noexcept(_LIBCPP_MOVE_ONLY_FUNCTION_NOEXCEPT);
+  using _DestroyFn = void(void*) noexcept;
 
   static constexpr size_t __buffer_size_ = 4 * sizeof(void*);
 
```

Comment:
(Bleah, I'm probably adding comments to the wrong GitHub entity, because it won't let me comment on line 247. Sorry.) Recommend eliminating the dependency on `<array>`.